crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -323,8 +323,23 @@ where
     /// If automatically inserting [`apply_deferred`](crate::schedule::apply_deferred) like
     /// this isn't desired, use [`after_ignore_deferred`](Self::after_ignore_deferred) instead.
     ///
-    /// Note: The given set is not implicitly added to the schedule when this system set is added.
-    /// It is safe, but no dependencies will be created.
+    /// Usually, you'll want to use [`.chain`](Self::chain) instead.
+    ///
+    /// # Caveats
+    ///
+    /// If you configure two [`SystemSet`]s like `(GameSystem::A).after(GameSystem::B)` or `(GameSystem::A).before(GameSystem::B)`, the `GameSystem::B` will not be automatically scheduled.
+    ///
+    /// This means that the systems in `GameSystem::A` and `GameSystem::B` will run independently of each other if `GameSystem::B` was never explicitly scheduled with [`App::configure_sets`].
```

Comment:
I don't think this is true, or maybe don't understand what this is saying. Configure_sets feels like a red herring.